### PR TITLE
 Add customKey & customFunction description

### PR DESCRIPTION
**Pull request by wkdtjsgur100**
- Add customKey option.
- Add cutomFunction & customKey description on README.md

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -122,6 +122,38 @@ new Fingerprint2({
...
+  console.log(result, components)
+});
+```
+
+##### `customKey` option
+
+`customKey` option allows you to change the key of the component. customKey must be string
+
+``` js
+new Fingerprint2({
```
Comment (jonashaag) [inline README.md]: Typo

**Comment by jonashaag** [comment]
LGTM, can you add a tiny test? Or adapt the customFunction test

**Comment by wkdtjsgur100** [comment]
@jonashaag Added!

**Comment by jonashaag** [comment]
<3

**Pull request merged**